api/api.py

```diff
@@ -27,11 +27,11 @@ def home():
 #TODO: Implement way to query DB and put all employees in an array like the employees array above to display in API response
 
 # A route to return all employees.
-@app.route('/api/v1/employees/all', methods=['GET'])
+@app.route('/api/v1/employees', methods=['GET'])
```

Comment:
Documentation also needs to be updated to reflect these changes. Other than that everything else looks good!